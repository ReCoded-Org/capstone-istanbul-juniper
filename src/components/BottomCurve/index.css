.bottomCurveContainer {
  background: #2a8c79;
}

.bottomCurveContainer__bottomCurve {
  background: #b4d0c4;
  border-radius: 50%;
  height: 15vh;
}

.bottomCurveContainer__bottomCurve__topHalf {
  height: 50%;
  background: #b4d0c4;
}

.bottomCurveContainer__bottomCurve__bottomHalf {
  height: 50%;
}
