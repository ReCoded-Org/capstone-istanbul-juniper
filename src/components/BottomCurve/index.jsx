/**
 * Navbar has a section that displays top half of a wide elliptical shape.
 * This component is the bottom half of elliptical shape
 */

import React from "react";
import "./index.css";

const BottomCurve = () => {
  return (
    <div className="bottomCurveContainer">
      <div className="bottomCurveContainer__bottomCurve">
        <div className="bottomCurveContainer__bottomCurve__topHalf" />
        <div className="bottomCurveContainer__bottomCurve__bottomHalf" />
      </div>
    </div>
  );
};
export default BottomCurve;
