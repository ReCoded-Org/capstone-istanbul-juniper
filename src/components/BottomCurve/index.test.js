import React from "react";
import renderer from "react-test-renderer";
import BottomCurve from ".";

it("BottomCurve renders correctly", () => {
  const tree = renderer.create(<BottomCurve />).toJSON();
  expect(tree).toMatchSnapshot();
});
