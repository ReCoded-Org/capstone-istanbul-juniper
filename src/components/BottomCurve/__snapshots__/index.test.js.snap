// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BottomCurve renders correctly 1`] = `
<div
  className="bottomCurveContainer"
>
  <div
    className="bottomCurveContainer__bottomCurve"
  >
    <div
      className="bottomCurveContainer__bottomCurve__topHalf"
    />
    <div
      className="bottomCurveContainer__bottomCurve__bottomHalf"
    />
  </div>
</div>
`;
